### PR TITLE
Squash

### PR DESCRIPTION
**Pull request by Maickonn**
Added squash action.

### CODE REVIEW

**Comment by Maickonn** [comment]
Hello, hackhaslam!

Can you review this pull request and tell me if you need to change anything to merge?

I'm waiting.

A big hug!

**Comment by Maickonn** [comment]
@hackhaslam you have any position about this pull request?

**Comment by hackhaslam** [comment]
I'm sorry. I haven't been able to review it yet. I will try to do it soon.

**Comment by Maickonn** [comment]
Okay, thanks.

File: src/ui/MenuBar.cpp [hunk 1]
```diff
@@ -481,6 +481,19 @@ MenuBar::MenuBar(QWidget *parent)
     dialog->open();
   });
 
+  mRebase = branch->addAction(tr("Squash..."));
+  mRebase->setShortcut(tr("Ctrl+Shift+S"));
```
Comment (tommai78101) [inline src/ui/MenuBar.cpp]: Quick question - Is CTRL+SHIFT+S an universal hotkey shortcut for most git frontend applications for a Squash action?

Comment (Maickonn) [reply src/ui/MenuBar.cpp]: Hi! Sorry for the delay. I searched and found no shortcut key patterns for squash.

**Comment by Maickonn** [comment]
@hackhaslam if possible, review this pull request. We need this function. 😅

File: src/ui/MenuBar.cpp (hunk 1 above)
Comment (tommai78101) [reply src/ui/MenuBar.cpp]: Very well, I think it's fair enough.

Comment (Maickonn) [reply src/ui/MenuBar.cpp]: Right, no problem.

Comment (hackhaslam) [reply src/ui/MenuBar.cpp]: You're resetting the `mRebase` member instead of setting the `mSquash` one you added.

**Comment by Maickonn** [comment]
Wow, sorry me!

I will fix it.

**Comment by hackhaslam** [comment]
@Maickonn Actually, maybe hold off a minute before making any changes. Is this meant to implement `merge --squash`? In that case, I don't know if it really merits its own top-level menu item.

**Comment by hackhaslam** [comment]
I merged this mostly as is. I fixed the menu bar issue I mentioned before, and changed the shortcut to Ctrl+Shift+Q since Ctrl+Shift+S is the typical shortcut for 'Save As'.

**Comment by Maickonn** [comment]
> I merged this mostly as is. I fixed the menu bar issue I mentioned before, and changed the shortcut to Ctrl+Shift+Q since Ctrl+Shift+S is the typical shortcut for 'Save As'.

It's all right. I'm sorry for not fixing the commit problem as you informed me in the review. I have been distracted for a while, but I will participate more actively.

Thank you for having accepted and corrected the pull request.

**Comment by hackhaslam** [comment]
Nah, it's alright. I didn't really expect you to fix that since I told you to hold off. I'm sorry it took me so long to circle back around to this.

**Comment by 0xCAP** [comment]
When is this feature gonna be included in a release? Latest update still didn't have it.

**Comment by Maickonn** [comment]
> When is this feature gonna be included in a release? Latest update still didn't have it.

Hello!

This is are available on latest update. Make right click on any branch, it will show below to the merge button.

**Pull request merged**